persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/impl/StargateSystemKeyspace.java

```diff
@@ -196,4 +203,20 @@ public static void forceBlockingFlush(String cfname) {
           Keyspace.open(SYSTEM_KEYSPACE_NAME).getColumnFamilyStore(cfname).forceFlush());
     }
   }
+
+  public static Set<String> generateRandomTokens(InetAddress inetAddress, int numTokens) {
+    Random random = new Random(getSeed(inetAddress));
```

Comment:
WDYT about `SecureRandom` ? We get a new seed every time, but the quality of the random sequence should be better with `SecureRandom`. WDYT?